docs/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "aztec-docs",
-  "version": "0.0.0",
+  "version": "0.83.0",
```

Comment:
should this be 83.1 now? Will this automatically update on new releases?

Reply:
yeah RP does this